.env.example

```diff
@@ -4,10 +4,10 @@ TWILIO_API_KEY=SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 TWILIO_API_SECRET=YYXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
 # Required for Chat
-TWILIO_CHAT_SERVICE_SID=ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+# TWILIO_CHAT_SERVICE_SID=ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
@ajtack This change is breaking the build.

At the moment I can only think of two possible solutions:

1. Modify [`config.js`](https://github.com/TwilioDevEd/sdk-starter-node/blob/master/src/config.js#L7) and do these changes:

   ```js
   TWILIO_CHAT_SERVICE_SID: process.env.TWILIO_CHAT_SERVICE_SID || 'default',
   ```

1. Revert this change and let it be as it was originally.

Greetings 👋

Reply:
Yep, saw this too late. `default` doesn't work for Chat instances, and `ISXXX` produces a broken demo, so I'll have to do something more clever.

Reply:
Build working. We now generate a .env that produces full tokens for testing, but all the .env.example lines are commented out to avoid a broken demo.

Reply:
Great job! 👍 thanks for tackling this issue!